chessington/ui/static/scripts/image_loader.js

```diff
@@ -0,0 +1,13 @@
+export async function loadPieceImage(piece, player) {
```

Comment:
Yes that's a nice visual improvement. Will need to swap out the `forEach` for a `for piece of pieces` loop with awaits to avoid a race condition on fetching the images.